bolt.ipynb

```diff
@@ -146,50 +90,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {
-    "scrolled": true
-   },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Loading train dataset...\n",
-      "Read 18100 vectors from datasets/intent_classification/train_shuf.svm in 2 seconds\n",
-      "Loading test dataset...\n",
-      "Read 5500 vectors from datasets/intent_classification/test_shuf.svm in 0 seconds\n"
-     ]
-    }
-   ],
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
    "source": [
-    "\n",
+    "# TODO(Geordie): Add download scripts and change to relative path\n",
     "print(\"Loading train dataset...\")\n",
     "intent_class_train = dataset.load_bolt_svm_dataset(filename=\"datasets/intent_classification/train_shuf.svm\", batch_size=256)\n",
     "\n",
     "print(\"Loading test dataset...\")\n",
     "intent_class_test = dataset.load_bolt_svm_dataset(filename=\"datasets/intent_classification/test_shuf.svm\", batch_size=256)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Sparse inference\n",
+    "You can also use sparsity to accelerate inference. Simply call the `enable_sparse_inference()` method. Notice that we call the method before the last training epoch. This freezes the hash functions, effectively locking specialized subnetworks for each input vector, and then fine-tunes these subnetworks."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "\n",
-      "Epoch 1:\n",
-      "[==================================================] 100%\n",
-      "Processed 71 training batches in 80 seconds\n",
-      "\n",
-      "Epoch 2:\n",
-      "[==                                                ] 4%"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "bigger_network.train(train_data=intent_class_train, loss_fn=bolt.CategoricalCrossEntropyLoss(), learning_rate=0.001, epochs=2)\n",
     "bigger_network.enable_sparse_inference()\n",
```

Comment:
I am a bit confused on why we would call `enable_sparse_inference()` here, since there is no call to `network.predict()` in this cell.